### ISSUE TITLE
Different Batch Sizes support 

### ISSUE THREAD

**Issue by nomaad42** [ENOT-AutoDL/onnx2torch#113]
Is it possible to convert the model from onnx to torch , and then run with different batch sizes?

**Comment by senysenyseny16** [comment]
Hi, @fengshuibuddy 

If ONNX model has dynamic batch size, PyTorch model will work with any batch size. 
If ONNX model has fixed batch size (especially 1), PyTorch model may not work with dynamic batch size; it depends on operators and optimizations of the ONNX model.

**Comment by fengshuibuddy** [comment]
Hi, @senysenyseny16 

Thank you for your response! Yeah, it's true, but I found one repo: 
https://github.com/ToriML/onnx2pytorch

In this repo, when you convert from ONNX to Torch model, you can choose the parameter "experimental=True", then the converted model can receive a dynamic batch size, but it can cause problems if there's a BatchNorm layer in the model. 

In addition, this repo can NOT convert all 'onnx' models, only some of them (it could not convert my ONNX model).
